graphql/schema.graphql

```diff
@@ -72,6 +80,11 @@ type Query {
   averageRatingByProductId(productId: String!): Float
     @cacheControl(scope: PRIVATE)
   totalReviewsByProductId(productId: String!): Int @cacheControl(scope: PRIVATE)
+  totalReviewsByProductId5(productId: String!): Int @cacheControl(scope: PRIVATE)
+  totalReviewsByProductId4(productId: String!): Int @cacheControl(scope: PRIVATE)
+  totalReviewsByProductId3(productId: String!): Int @cacheControl(scope: PRIVATE)
+  totalReviewsByProductId2(productId: String!): Int @cacheControl(scope: PRIVATE)
+  totalReviewsByProductId1(productId: String!): Int @cacheControl(scope: PRIVATE)
```

Comment:
We can remove those queries and just add the new field `totals` to `ReviewsResult`
```suggestion
```